components/channel_header/channel_header.js

```diff
@@ -231,6 +231,10 @@ export default class ChannelHeader extends React.PureComponent {
         }
     }
 
+    removeTooltipLink = () => {
+        this.refs.toggleFavorite.removeAttribute('aria-describedby');
```

Comment:
Why are we removing an attribute that doesn't exists?

Reply:
@jespino Bootstrap adds it in when the tooltip opens up, so removing it so that the button is no longer to the tooltip in the accessibility readout.

Reply:
I would add a comment explaining that right there, is a not obvious at all from the code, actually, If I see that code, I would remove it because is doing something unnecessary (from the POV of an isolated chunk of code).

Reply:
Another thing, would be nice to use here the new `React.createRef` approach (like you used in the "sidebar channel button or link")

Reply:
Done.